goodtablesio/tests/integrations/s3/models/test_s3bucket.py

```diff
@@ -90,3 +93,18 @@ def test_secret_access_key_del():
     bucket_db = database['session'].query(S3Bucket).first()
 
     assert bucket_db.secret_access_key is None
+
+
+# https://github.com/frictionlessdata/goodtables.io/issues/192
```

Comment:
I guess that what the test could test is that if the user has another unrelated source (eg `factories.GithubRepo()`) it is still present after deleting the bucket (`user.sources[0] == github_repo`), as that's what #192 was causing